lib/elixir/lib/io/ansi.ex

```diff
@@ -333,4 +336,12 @@ defmodule IO.ANSI do
   defp do_format([], [], acc, _emit?, _append_reset) do
     acc
   end
+
+  defp ensure_valid_ansi_sequecne(sequence) do
+    try do
+      format_sequence(sequence)
+    rescue
+      error in ArgumentError -> IO.warn(error.message)
```

Comment:
I don't think we should convert it to a warning. Or it consistently raises or we ignore it. :)

Reply:
Updated :)